### DISCUSSION TITLE
Windows 2.03 Works

### DISCUSSION THREAD

**Discussion by ghost** [copy/v86#515] [Show and tell]
Windows 2.03 Works on v86! but some times it freezes if i open too many applications
[Download Floppy Image Here](https://github.com/copy/v86/files/7001620/windows2_floppy.zip)

![image](https://user-images.githubusercontent.com/71384075/129766386-6c3031de-10e9-4a9c-bd11-5f2d0b6db7e3.png)

**Comment by ghost** [comment]
when i opened Write.exe , its crashed

**Comment by Pixelsuft** [comment]
Here is an [hard disk image](https://pixelsuftbox.herokuapp.com/static/img/win2.img)
